src/GovUk.Education.ExploreEducationStatistics.Admin/GovUk.Education.ExploreEducationStatistics.Admin.csproj

```diff
@@ -201,7 +201,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="Migrations\ContentMigrations" />
     <Folder Include="Migrations\UsersAndRolesMigrations" />
```

Comment:
Was this file change intentional?

Reply:
Rider seems to remove this as soon as I add a new file to the directory. It doesn't matter whether that's a c# `.cs` file or any other type of file.

If I add a new directory I get a new `ItemGroup/Folder` entry in the `.csproj` file up until the point I add files to the directory. Presumably this is to stop it from being lost without any files existing under version control in Git if the project was to be restored from a clean checkout.

Do you know if we need to keep it for any reason?

Reply:
I've no idea I'm afraid - if it's not causing any issues and the migrations are being applied, I guess potentially we don't need it - should we remove the line underneath too?

Reply:
Yes I think we can, I've removed the entries which I think are unnecessary now.